module/Application/view/error/404.phtml

```diff
@@ -52,7 +52,8 @@ if (! empty($this->reason)) :
 
 <?php if (! empty($this->display_exceptions)) : ?>
 
-    <?php if (isset($this->exception) && $this->exception instanceof Exception) : ?>
+    <?php if (isset($this->exception)
+        && ($this->exception instanceof \Exception || $this->exception instanceof \Error)) : ?>
```

Comment:
Elsewhere we check first for Throwable, than Exception, which optimizes for PHP 7 (where both exception and errors are types of Throwable). I'll make that change on merge.